### DISCUSSION TITLE
Animetrix is back live

### DISCUSSION THREAD

**Discussion by ShivaBhattacharjee** [ShivaBhattacharjee/AnimeTrix-next#14] [Announcements]
Well after probably a month or two of inactivity animetrix is live again sorry for the inconvenience

**Comment by ImmortalXnetwork** [comment]
player dead cant able to play videos
plz resolve it

**Reply by ShivaBhattacharjee** [reply]
![Screenshot_20240621-125652.png](https://github.com/ShivaBhattacharjee/AnimeTrix-next/assets/95211406/ec438478-e064-407d-932a-5fc2b2cb314f)

Working fine
